### PR TITLE
get identity id: try to read from os environment

### PR DESCRIPTION
**Pull request by LiliDeng**

### CODE REVIEW

File: lisa/sut_orchestrator/azure/common.py [hunk 1]
```diff
@@ -2734,6 +2734,9 @@ def get_managed_identity_object_id(
 def get_identity_id(
     platform: "AzurePlatform", application_id: Optional[str] = None
 ) -> Any:
+    if not application_id:
+        application_id = os.environ.get("AZURE_CLIENT_ID")
```
Comment (squirrelsc) [inline lisa/sut_orchestrator/azure/common.py]: Make sure it won't fail, if the variable doesn't exist in the environment, like `os.environ.get("AZURE_CLIENT_ID", "")`

File: microsoft/testsuites/vm_extensions/azure_keyvault_vm_extension.py [hunk 2]
```diff
@@ -63,7 +63,7 @@ def _check_system_status(node: Node, log: Logger) -> None:
         expected_exit_code_failure_message=message,
     )
 
-    certb = "/var/lib/waagent/a/symbolinkb"
+    certb = "/var/lib/waagent/b/symbolinkb"
```
Comment (squirrelsc) [inline microsoft/testsuites/vm_extensions/azure_keyvault_vm_extension.py]: Why is this char changed? Is it possible to be changed in future? Should we use a wildchar to match more paths?

Comment (LiliDeng) [reply microsoft/testsuites/vm_extensions/azure_keyvault_vm_extension.py]: it is a test bug, it was written wrong in the beginning.
image

Comment (squirrelsc) [reply microsoft/testsuites/vm_extensions/azure_keyvault_vm_extension.py]: Should it support both a and b?

File: lisa/sut_orchestrator/azure/common.py (hunk 1 above)
Comment (LiliDeng) [reply lisa/sut_orchestrator/azure/common.py]: updated

File: microsoft/testsuites/vm_extensions/azure_keyvault_vm_extension.py (hunk 2 above)
Comment (LiliDeng) [reply microsoft/testsuites/vm_extensions/azure_keyvault_vm_extension.py]: updated

**Review by squirrelsc — APPROVED** [review]

**Pull request merged**